README.md

```diff
@@ -218,6 +219,112 @@ You can also view the updated Application spec by going to this address: https:/
 The second package directory defines two normal ArgoCD applications referencing a remote repository.
 They are applied as-is.
 
+## Local OCI Registry
+
+The local Gitea instance created by idpbuilder contains a built in OCI registry for hosting container images as "packages" in Gitea nomenclature.
+
+It is a standard OCI registry, so the API should be compatible with any tools that are OCI compliant. That includes the `docker` cli.
+
```

Comment:
Make a warning here that to login username is case sensitive `giteaAdmin` but when using username/org in OCI image url needs be lower case `giteaadmin` (noticed there are 2 `a`)

I use `giteaAdmin` in docker tag, and lost a lot of time why is not working

Reply:
this is a good call. not sure why we need to call it `giteaAdmin` where `admin` would be good enough. I will create an issue for it so we can change it.